docs/upgrade/v1-0-3-to-v1-1-1.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 2
+sidebar_label: Upgrade to v1.1.1
+title: "Upgrade from v1.0.3/v1.1.0 to v1.1.1"
+---
+
+
+
+## General information
+
+:::caution
+
+- Starting from version v1.1.0, Harvester brings in the new [VLAN enhancement](https://github.com/harvester/harvester/issues/2236) feature. Due to the implementation changes, all user VMs must shut down if the upgrade is from v1.0.3 to v1.1.1. Please stop the VMs before an upgrade. Upgrading from v1.1.0 to v1.1.1 doesn't have this restriction.
+
+:::
+
+Once there is an upgradable version, the Harvester GUI Dashboard page will show an upgrade button. For more details, please refer to [start an upgrade](./automatic.md#start-an-upgrade).
+
+For the air-gap env upgrade, please refer to [prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
+
+
+## Known issues
+
+---
+
+### 1. An upgrade is stuck when pre-draining a node
+
+Starting from v1.1.0, Harvester will wait for all volumes to become healthy (when node counts >= 3) before upgrading a node. Generally, users can check volumes' health if an upgrade is stuck in the "pre-draining" state.
+
+Visit ["Access Embedded Longhorn"](../troubleshooting/harvester.md#access-embedded-longhorn) to see how to access the embedded Longhorn GUI.
+
+---
+
+### 2. An upgrade is stuck when pre-draining a node (case 2)
+
+An upgrade is stuck, as shown in the screenshot below:
+
+![](/img/v1.1/upgrade/known_issues/3216-stuck-pre-drain.png)
+
+And the user can also observe multiple nodes have scheduling disabled.
+
+```
+$ kubectl get nodes
+NAME    STATUS                     ROLES                       AGE   VERSION
+node1   Ready                      control-plane,etcd,master   20d   v1.24.7+rke2r1
+node2   Ready,SchedulingDisabled   control-plane,etcd,master   20d   v1.24.7+rke2r1
+node3   Ready,SchedulingDisabled   control-plane,etcd,master   20d   v1.24.7+rke2r1
+```
+
+- Related issue:
+  - [[BUG] Multiple nodes pre-drains in an upgrade](https://github.com/harvester/harvester/issues/3216)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/3216#issuecomment-1328607004
+
+---
+
+### 3. Upgrade stuck in upgrading first node: Job was active longer than specified deadline
```

Comment:
```suggestion
### 3. An upgrade is stuck in upgrading the first node: Job was active longer than the specified deadline
```

Reply:
I changed this sentence only to keep the structure the same as other counterparts~